### PR TITLE
New package: nvme-cli-1.4

### PR DESCRIPTION
**Pull request by phf**
It's my first attempt at putting a package in, please feel free to tell me all the things that are wrong with this.

### CODE REVIEW

File: srcpkgs/nvme-cli/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=nvme-cli
+version=1.4
+revision=1
+build_style=gnu-makefile
+make_install_args="SBINDIR=/usr/bin"
+short_desc="NVMe management command line interface"
+maintainer="Peter Froehlich <peter.hans.froehlich@gmail.com>"
+license="GPL-2"
+homepage="https://github.com/linux-nvme/nvme-cli"
+distfiles="https://github.com/linux-nvme/nvme-cli/archive/v1.4.tar.gz"
```
Comment (pullmoll) [inline srcpkgs/nvme-cli/template]: Use `.../v${version}.tar.gz` here.

Comment (cr6git) [reply srcpkgs/nvme-cli/template]: `homepage="http://nvmexpress.org"`

**Comment by phf** [comment]
Thanks for the feedback! (I have to admit that the "push -f" instructions freaked me out a little, but then again the detailed history of these is maybe not very important.)

**Comment by phf** [comment]
It does seem that the "push -f" breaks how github tracks this PR. @pullmoll I wonder if you need to confirm that the changes I made are what you wanted? (Sorry, not too familiar with this process.)

**Comment by cr6git** [comment]
@phf No problem with your `git push -f` , and the changes you made are correct.

File: srcpkgs/nvme-cli/template (hunk 1 above)
Comment (phf) [reply srcpkgs/nvme-cli/template]: Don't want to go on your nerves, but those are fixed. Any other reason holding back the merge? I am happy to work on it!

**Pull request closed without merging**